packages/typespec-ts/src/modular/helpers/typeHelpers.ts

```diff
@@ -38,7 +38,7 @@ export function getType(type: Type, format?: string): TypeMetadata {
     case "boolean":
       return { name: getNullableType(type.type, type) };
     case "constant": {
-      let typeName: string = type.value ?? "undefined";
+      let typeName: string = type.value?.toString() ?? "undefined";
```

Comment:
What is the difference with and without toString()?

Reply:
If the type is literal type like true, what we get for typeName is not a string but a true value. though it's claimed to be a string, but it doesn't do convert by itself.